phpBB/posting.php

```diff
@@ -91,7 +89,6 @@
 	'topic_id',
 	'forum_id',
 	'draft_id',
-	'lastclick',
```

Comment:
Please add a `@changed` tag similar to [this one](https://github.com/phpbb/phpbb/blob/release-3.2.2/phpBB/posting.php#L87) to the doc-block.